guides/source/testing.md

```diff
@@ -1439,7 +1439,7 @@ class UsersTest < ApplicationSystemTestCase
   # test "visiting the index" do
   #   visit users_url
   #
-  #   assert_selector "h1", text: "Users"
+  #   assert_select "h1", text: "Users"
```

Comment:
The ApplicationSystemTestCase does implement `assert_selector` as it includes Capybara assertions.
```suggestion
  #   assert_selector "h1", text: "Users"
```

Reply:
so that's it